parsec/core/gui/files_widget.py

```diff
@@ -764,7 +771,35 @@ def _on_import_success(self):
         self.import_job = None
 
     def _on_import_error(self):
+        def _display_import_error(file_count, exceptions=None):
+            print("WTF", exceptions, file_count)
```

Comment:
print still usefull ?
If it is for the CLI, maybe changing the text for something like "Import files failed"